.github/workflows/test.yml

```diff
@@ -0,0 +1,65 @@
+name: ci
+
+on:
+  - push
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install package
+        run: pip install .[dev]
+
+      - name: Run commit-blocking lint checks
+        run: pre-commit run --all-files
+
+  unit-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install package
+        run: pip install .[dev]
+
+      - name: Run unit tests
+        run: pytest --log-format="%(asctime)s %(levelname)s %(message)s" --log-date-format="%Y-%m-%d %H:%M:%S"
+
+  integration-test:
+    runs-on: ubuntu-latest
+    concurrency: integration-test
```

Comment:
oh my gosh! I didn't know about this feature. I'm going to start using it for integration tests too!

Reply:
Looks like it only came out a few months ago.  But "Note: Concurrency is currently in beta and subject to change."...